tests/unit/Registry.ts

```diff
@@ -97,57 +96,50 @@ registerSuite({
 				assert.isNull(item);
 			},
 			'replaces promise with result on resolution'() {
-				let resolveFunction: any;
-				const promise: any = new Promise((resolve) => {
-					resolveFunction = resolve;
+				const promise: Promise<any> = new Promise((resolve) => {
+					resolve(WidgetBase);
 				});
+
 				const factoryRegistry = new Registry();
 				factoryRegistry.define('my-widget', promise);
 				factoryRegistry.get('my-widget');
-				resolveFunction(WidgetBase);
```

Comment:
Isn’t the point of this test to resolve after the get?

Reply:
True. I'll fix this and the next test.